CSharp/ArticleProjects/EfficientListQueries/Program.cs

```diff
@@ -0,0 +1,273 @@
+﻿// Copyright (c) Microsoft Corporation
+//
+// Companion project backing the code snippets found in the following article:
+// https://azure.microsoft.com/documentation/articles/batch-efficient-list-queries/
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Azure.Batch;
+using Microsoft.Azure.Batch.Auth;
+using Microsoft.Azure.Batch.Common;
+using Microsoft.Azure.Batch.Samples.Common;
+
+namespace Microsoft.Azure.Batch.Samples.Articles.EfficientListQueries
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            // Each task that we create below is intentionally quite large (>2048 bytes) so

```

Comment:
saying a task is some number of bytes doesn't make sense. Must be an attribute of a task so better to state that.